api/src/components/sitemap/service.ts

```diff
@@ -62,32 +69,42 @@ export const generatePublicationSitemaps = async (): Promise<void> => {
         await s3.client.send(
             new PutObjectCommand({
                 Bucket: s3.buckets.sitemaps,
-                Key: `publications/${sitemapCount}.xml`,
+                Key: `${category}/${sitemapCount}.xml`,
                 ContentType: 'text/xml',
                 Body: sitemap
             })
         );
-        queryResults = await client.prisma.publication.findMany({
-            skip: 1,
-            cursor: {
-                id: queryResults[queryResults.length - 1].id
-            },
-            ...queryOptions
-        });
+        queryResults =
```

Comment:
I know this is slightly out of scope for this PR, but this method of getting the publications/users is a bit odd.
Get 50k, do something with them, then get the next 50k, do something with them etc etc.

Would it make more sense to get the lot, split into 50k buckets and dispatch the buckets off to google one by one? I've done a similar thing here - https://github.com/JiscSD/github-monitor/blob/d9221383d80ef52b26ae6710d5fecd5adf9a6cd1/src/utils/DynamoStore.ts#L14 - gets all data, splits into 25 item chunks (dynamodb limit) then loops through chunks saving them to database.

Only potential issue could be the sheer amount of sitemaps coming from database..

Reply:
Yeah either way works I guess. My preference is always to limit the amount of round trips to database, but tradeoff is inefficiency of javascript. I guess my example doesn't have >50k entries to process 🙃